likelihood/cl_like/lpt.py

```diff
@@ -52,9 +52,10 @@ def update_pk(self, pk, Dz):
                               kmax=self.ks[-1]/self.h, nk=self.ks.size)
             self.lpt_table.append(cleft.pktable)
         self.lpt_table = np.array(self.lpt_table)
-        self.lpt_table /= self.h**3
+        self.lpt_table[:, :, 1:] /= self.h**3
```

Comment:
what's going on here?

Reply:
The zero-component of this is k, so we don't really want to divide it by h^3.